src/utils/csv.ts

```diff
@@ -0,0 +1,35 @@
+import { writeToString } from "@fast-csv/format";
+
+type RowInterface = {
+  [key: string]: string;
+};
```

Comment:
This is basically just a `Record<string, string>`

Reply:
fixed in a2c3ebfeb0bd6fc1d4c317f56172510fc58adcab